### ISSUE TITLE
How to add All price types in my product detail for choosing it?

### ISSUE THREAD

**Issue by Emilianissimo** [oc-shopaholic/oc-shopaholic-plugin#254]
I have a big trouble with understanding this. I can't save one of Offer's price type into the database and when I make this, I have an error:

> function onInit() {
>     PriceTypeHelper::instance()->switchActive('b2b');
> }

Error is like: undefined PriceTypeHelper.
So, I make a forloop inside my product-detail part with Offers and it's giving me all my price types, but when i push the button, it's not sending into my cart. I made it like select tag with options and value, but didn;t found where I can catch it and other things.
I just want to make choice for my price types inside product-detail for my customers. Thanks a lot, sorry if it look like daydreamer's message, i wrote it at 3 am and nearly died by looking inside docs for my issue.

**Comment by kharanenka** [comment]
Hi!
 I think error arose because you did not attach PriceTypeHelper class.
```php
use Lovata\Shopaholic\Classes\Helper\PriceTypeHelper;
```

If you want to create orders with different price types, then you should switch the active price type before creating an order. You cannot create an order in which offers with different types of prices.
You can create an order with different price types. But all offers must be with same price type in one order.

**Comment by Emilianissimo** [comment]
Thanks how answer, but where i need to put this using? In function onInit()? And what you mean by different prices and one type? One type has one price as i know

**Comment by kharanenka** [comment]
```php
PriceTypeHelper::instance()->switchActive('b2b');
```

This code is best located in Plugin.php file, boot() method

**Comment by Emilianissimo** [comment]
Here is first offer
![image](https://user-images.githubusercontent.com/48443034/75229303-77550d00-57d3-11ea-9541-465686768a2d.png)
I made two price types
![image](https://user-images.githubusercontent.com/48443034/75229228-542a5d80-57d3-11ea-8592-fe9f98b1ac65.png)
Added your method into the Plugin.php in the both plugins (Shopaholic and Orders). How could I display that for switching in my Product?
Here the Product
![image](https://user-images.githubusercontent.com/48443034/75229432-b4210400-57d3-11ea-9310-11e657f14925.png)
Here the cart
![image](https://user-images.githubusercontent.com/48443034/75229458-c0a55c80-57d3-11ea-86a9-704bb2af2764.png)

Sorry for disturbing, but it's very important for me.
